acme/setup.py

```diff
@@ -28,6 +27,13 @@
     'six',
 ]
 
+# env markers cause problems with older pip and setuptools
+if sys.version_info < (2, 7):
+    install_requires.extend([
+        'argparse',
```

Comment:
Yes. I put this in my [original post](https://github.com/certbot/certbot/pull/4449#issue-218398044), but I rebased this PR on #4554.